### PR TITLE
Confimation delete post and comment popup

### PR DESCRIPTION
**Pull request by mvn-tamnguyen4-dn**
## Description
Confimation delete post and comment popup 
## Reference Requirements
- Product Backlog ID: PBI-11
- Task ID: 2

## Reference Isuees
None

## Migrations
 NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

- Helper/common.php
- posts/show.blade.php

## SQL query and `explain`
None

### CODE REVIEW

File: resources/views/backend/posts/show.blade.php [hunk 1]
```diff
@@ -1,6 +1,55 @@
...
+  <div class="modal-dialog">
+
+    <!-- Modal content-->
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">&times;</button>
+      </div>
+      <div class="modal-body text-center">
+        <h3>{{ __('post.confirm.title') }}</h3>
+        <p >{{ __('post.confirm.delete_post') }} ?</p>
```
Comment (at-minhphan) [inline resources/views/backend/posts/show.blade.php]: dư space

File: resources/views/backend/posts/show.blade.php [hunk 2]
```diff
@@ -1,6 +1,55 @@
...
+  <div class="modal-dialog">
+
+    <!-- Modal content-->
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">&times;</button>
+      </div>
+      <div class="modal-body text-center">
+        <h3>{{ __('post.confirm.title') }}</h3>
+        <p >{{ __('post.confirm.delete_comment') }} ?</p>
```
Comment (at-minhphan) [inline resources/views/backend/posts/show.blade.php]: dư space

**Comment by at-minhphan** [comment]
LGTM

**Comment by at-duyho** [comment]
- fix conflict and all comments. thank!

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**